easypost.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
   spec.required_ruby_version = '>= 2.6'
 
+  spec.add_development_dependency 'activesupport', '~> 6.1.7.3'
```

Comment:
question: why are we introducing `activesupport`? It is traditionally a Ruby on Rails dependency, since this is a library, we shouldn't be introducing it here. We've gone this far without it. If you are using an active support function, I'd suggest finding the vanilla Ruby equivalent. I'd prefer we don't add a large framework to the library.

Reply:
We need `activesupport` for the `constantize` method in the `base.rb` file. This method is part of the `ActiveSupport` https://apidock.com/rails/String/constantize

Reply:
We should meet to discuss this specifically, I have a lot of thoughts.